src/OrleansSQLUtils/OrleansSQLUtils.csproj

```diff
@@ -13,6 +13,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Orleans.Core.Abstractions\Orleans.Core.Abstractions.csproj" />
+    <ProjectReference Include="..\OrleansRuntime\OrleansRuntime.csproj" />
```

Comment:
Hopefully once we split the packages, we can have client one that doesn't require a reference to the silo package, right @attilah?

Reply:
It should be possible, I'd really like to explode Orleans.Core after abstractions into Orleans.Client and Core, that would be really useful if client would not be only a meta package.